src/components/layouts/EmailSentSuccess/index.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {connect} from 'react-redux'
+import __spacing from 'settings/__spacing';
+import __layouts from 'settings/__layouts';
+import { fontSizes} from 'settings/__fonts';
+
+const EMAIL_IMAGE = "https://trello-attachments.s3.amazonaws.com/5db1b02b5b9a3e413fb742f2/5ddd0139b88cae26568ed400/ac49" +
+    "dbf3c29bc291362a3ca14c393b77/undraw_Mail_sent_qwwx.svg";
+
+function EmailSentSuccess({navigationData}) {
+    const name = navigationData.userData? navigationData.userData.firstName: 'there!'
+    const SIGNUP_MESSAGE =  `Hi ${name} your signup was  successful and a verification
+        mail was sent to your email. Please confirm your email to continue`;
+     const RESET_PASSWORD_MESSAGE =
+         `Hi there! Instructions to reset your password has been sent to your email.
+         Please check it out to reset your password thanks!`;
+  return (
+    <EmailSentSuccess.Wrapper>
```

Comment:
This should be a component in the`UI` folder for now and not a layout. It's page uses the `AuthenticationLayout`.

Reply:
done. Also decided not to connect `EmailSentSuccess` to store. Instead sent down props from `AuthFeedback`